e2e-tests/gef/cypress/integration/page-banner.spec.js

```diff
@@ -0,0 +1,53 @@
+import relative from './relativeURL';
+import CREDENTIALS from '../fixtures/credentials.json';
+import pageBanner from './pages/page-banner';
+
+context('Check GEF GOVUK banner displays correctly', () => {
```

Comment:
can we call this file and page **header** rather than banner?

Reply:
done